service/src/ocall_bridge/component_factory.rs

```diff
@@ -17,6 +17,7 @@
 */
 
 use crate::{
+	globals::tokio_handle::GetTokioHandle,
```

Comment:
this was previously exported to the crate root in main, I thought this is not necessary actually.